generated/nidcpower/visatype.py

```diff
@@ -6,69 +6,19 @@
 '''
 
 
-class ViStatus(ctypes.c_long):  # noqa: N801
-    pass
+ViStatus = ctypes.c_long
+ViRsrc = ctypes.c_char_p
+ViSession = ctypes.c_ulong
+ViChar = ctypes.c_char
+ViUInt32 = ctypes.c_ulong
+ViInt32 = ctypes.c_long
+ViInt16 = ctypes.c_short
+ViUInt16 = ctypes.c_ushort
+ViInt64 = ctypes.c_longlong
+ViString = ctypes.c_char_p
+ViAttr = ctypes.c_long
+ViConstString = ViString
+ViBoolean = ctypes.c_ushort
+ViReal32 = ctypes.c_float
+ViReal64 = ctypes.c_double
 
-
-class ViRsrc(ctypes.c_char_p):  # noqa: N801
-    pass
-
-
-class ViSession(ctypes.c_ulong):  # noqa: N801
-    pass
-
-
-class ViChar(ctypes.c_char):  # noqa: N801
-    pass
-
-
-class ViUInt32(ctypes.c_ulong):  # noqa: N801
-    pass
-
-
-class ViInt32(ctypes.c_long):  # noqa: N801
-    pass
-
-
-class ViInt16(ctypes.c_short):  # noqa: N801
-    pass
-
-
-class ViUInt16(ctypes.c_ushort):  # noqa: N801
-    pass
-
-
-class ViInt64(ctypes.c_longlong):  # noqa: N801
-    pass
-
-
-class ViString(ctypes.c_char_p):  # noqa: N801
-    pass
-
-
-class ViAttr(ctypes.c_long):  # noqa: N801
-    pass
-
-
-class ViConstString(ViString):  # noqa: N801
-    @property
-    def value(self):  # Makes 'value' readonly
-        return super(ViConstString, ViString).value
-
-
-class ViBoolean(ctypes.c_ushort):  # noqa: N801
-    @classmethod
-    def from_param(cls, param):
-        return ctypes.c_uint16(1) if bool(param) else ctypes.c_uint16(0)
```

Comment:
We no longer have this.
I suppose ctypes just handled all the conversion from and to True and False?